pages/linux/goaccess.md

```diff
@@ -0,0 +1,20 @@
+# goaccess
+
+> An open source real-time web log analyzer.
+> More information: <https://goaccess.io>.
+
+- Run in interactive mode:
+
+`goaccess {{path/to/file}} [{{path/to/file}}]`
+
+- Use a specific log-format (or pre-defined formats like "combined"):
+
+`goaccess {{path/to/file}} --log-format={{format}}`
+
+- Analyse a log from stdin:
+
+`tail -f {{path/to/logfile}} | goaccess -`
+
+- Realtime HTML output in a single file (you can even mail this):
```

Comment:
```suggestion
- Analyze a log and write it to an HTML file in real-time:
```
What do you mean by `mailing`? Does the command have a flag for that? Because if yes, that should be in a separate example.

Reply:
It creates a single html file, which makes it perfect for sending it by mail. There is no special mail-option :(